seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -1133,8 +1133,9 @@
       ],
       "tests": [
         "assert(!/\\\\/g.test(code) && myStr === '<a href=\"http://www.example.com\" target=\"_blank\">Link</a>', 'message: Remove all the <code>backslashes</code> (<code>\\</code>)');",
-        "assert(code.match(/\"/g).length === 4 && code.match(/'/g).length === 2, 'message: You should have two single quotes <code>&#39;</code> and four double quotes <code>&quot;</code>');"
+        "assert((function() { const strict = \"'use strict';\"; const tail = '(function() { return \"myStr = \" + myStr; })();'; const sliced = code.slice(code.indexOf(strict) + strict.length, code.indexOf(tail)); if (sliced.match(/\"/g).length === 4 && sliced.match(/'/g).length === 2) return true; return false; })() === true, 'message: You should have two single quotes <code>&#39;</code> and four double quotes <code>&quot;</code>');"
```

Comment:
This solution assumes that the tail in the seed-file matches the tail that right now is hardcoded in the test. It could work, but it could also cause problems if we're unlucky.

A possible solution could be to just use the start of the tail section as argument for `code.indexOf(tail)`. If we added the comment `/*Required for test. Do not remove.*/` to the beginning of the tail `IIFE`, we could use this "constant string" to decide where the tail starts without having to rely on the exact code of the tail section.